core-bundle/src/Resources/contao/dca/tl_preview_link.php

```diff
@@ -106,21 +106,19 @@
 			'eval'                    => array('mandatory'=>true, 'readonly'=>true, 'rgxp'=>'url', 'decodeEntities'=>true, 'maxlength'=>2048),
 			'sql'                     => "varchar(2048) NOT NULL default ''",
 		),
-		'expiresInDays' => array
+		'showUnpublished' => array
 		(
 			'filter'                  => true,
-			'inputType'               => 'select',
-			'options'                 => array('1', '7', '30'),
-			'reference'               => &$GLOBALS['TL_LANG']['tl_preview_link']['expire_options'],
-			'eval'                    => array('mandatory'=>true, 'submitOnChange'=>true, 'tl_class'=>'w50'),
-			'sql'                     => "int(10) unsigned NOT NULL default 1"
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => "char(1) NOT NULL default ''"
```

Comment:
```suggestion
			'sql'                     => "char(1) NOT NULL default '1'"
```

Shouldn‘t the "show unpublished" option be checked by default, too? This is the primary use case for preview links IMHO.

Reply:
I think I’d keep it that way to make sure people explicitly allow that if they want to. 
We also shouldn’t change this in a bugfix release I think.